bin/gwdetchar-scattering

```diff
@@ -49,6 +51,8 @@ from gwdetchar.io import html as htmlio
 from gwdetchar.io.datafind import get_data
 from gwdetchar.plot import texify
 from gwdetchar.omega import batch
+from gwdetchar.scattering import core
+from gwdetchar.io import html as htmlio
```

Comment:
@siddharth101, these imports aren't needed either. The `html` module is already imported as `htmlio`, and this line:

```python
from gwdetchar import (cli, const, scattering)
```

already imports the `scattering` module, which already includes everything in `core`. Thus, you can globally replace every reference to `core` with a reference to `scattering` in this file, e.g. `scattering.get_blrms` rather than `core.get_blrms`.